### PR TITLE
refactor: allow issue-key in title or body

### PR DESCRIPTION
**Pull request by cyrus-za**
Allow PR title and/or body to be used to get JIRA issue from

### CODE REVIEW

**Comment by cyrus-za** [comment]
I did commit the built index.js file.

It might be a git setting of line-endings or something.

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 14.14.0
```
Comment (maddhruv) [inline .tool-versions]: what is this file here for?
can you please uncommit or remove this file?

Comment (ivorsmorenburg) [inline .tool-versions]:
```suggestion
```

Comment (cyrus-za) [reply .tool-versions]: removed

**Review by ivorsmorenburg — COMMENTED** [review]
LGTM

**Pull request closed without merging**